Gemfile

```diff
@@ -7,6 +7,8 @@ gem 'rake'
 gem 'rspec'
 gem 'coveralls', require: false
 
+gem 'redis', '<= 4.0.3' if RUBY_VERSION < '2.3'
```

Comment:
I need to do it like this because the documented one
```
gem "redis", '<= 4.0.3', platforms: :ruby_22
```
was not working... not sure why